README.md

```diff
@@ -1,258 +1,103 @@
-[![Build Status](https://travis-ci.org/cloudinary/cloudinary_npm.svg?branch=master)](https://travis-ci.org/cloudinary/cloudinary_npm)
+Cloudinary Node SDK
+=========================
+## About
+The Cloudinary Node SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize, transform, upload and manage your cloud's assets.
 
-Cloudinary
-==========
 
-Cloudinary is a cloud service that offers a solution to a web application's entire image management pipeline.
+#### Note
+This Readme provides basic installation and usage information. 
+For the complete documentation, see the [Node SDK Guide](https://cloudinary.com/documentation/node_integrations).
 
-Easily upload images to the cloud. Automatically perform smart image resizing, cropping and conversion without installing any complex software. Integrate Facebook or Twitter profile image extraction in a snap, in any dimension and style to match your website’s graphics requirements. Images are seamlessly delivered through a fast CDN, and much much more.
+## Table of Contents
+- [Key Features](#key-features)
+- [Version Support](#Version-Support)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Transform and Optimize Assets](#Transform-and-Optimize-Assets)
+    - [Generate Image and HTML Tags](#Generate-Image-and-Video-HTML-Tags)
 
-Cloudinary offers comprehensive APIs and administration capabilities and is easy to integrate with any web application, existing or new.
 
-Cloudinary provides URL and HTTP based APIs that can be easily integrated with any Web development framework.
+## Key Features
+- [Transform](https://cloudinary.com/documentation/node_video_manipulation#video_transformation_examples) and
+ [optimize](https://cloudinary.com/documentation/node_image_manipulation#image_optimizations) assets.
+- Generate [image](https://cloudinary.com/documentation/node_image_manipulation#deliver_and_transform_images) and
+ [video](https://cloudinary.com/documentation/node_video_manipulation#video_element) tags.
+- [Asset Management](https://cloudinary.com/documentation/node_asset_administration).
+- [Secure URLs](https://cloudinary.com/documentation/video_manipulation_and_delivery#generating_secure_https_urls_using_sdks).
 
-For Node.js, Cloudinary provides an extension for simplifying the integration even further.
 
-## Getting started guide
-![](https://res.cloudinary.com/cloudinary/image/upload/see_more_bullet.png)  **Take a look at our [Getting started guide for Node.js](https://cloudinary.com/documentation/node_integration#node_js_getting_started_guide)**.
 
+## Version Support
+| SDK Version   | node 6-14 |
+|---------------|-----------|
+| 1.28.1 & up    | X        |
```

Comment:
1. Show be a V instead of an X
2. What about Node 16?

Reply:
updated. Are tests only run on 6-14 so officially this is what we support